### DISCUSSION TITLE
How to not trigger deployment when files outside of root directory are pushed to master?

### DISCUSSION THREAD

**Discussion by Amberlamps** [vercel/vercel#6102] [Help]
I have to folders in my repo:

```
/api
/app
```

In my `/app` folder I have my next application. The `/api` folder is a serverless.com application and completely unrelated to my next application. However every time I push a change to the `/api` to Github the Vercel integration runs and rebuilds my next application.

How can I tell Vercel to only build & deploy when files from the `/app` folder change?

<img width="759" alt="Screenshot 2021-04-12 at 23 53 21" src="https://user-images.githubusercontent.com/1527079/114467501-4b206980-9bea-11eb-8bd7-fb9b832371ed.png">

**Comment by rubenkaiser** [comment]
https://github.com/vercel/vercel/discussions/5676
